src/javascripts/test/e2e/filterViewSpec.js

```diff
@@ -165,7 +165,7 @@ describe('List filter', function () {
             $$('.filters .filter:nth-child(1) input').sendKeys('be');
             browser.driver.sleep(600); // debounce delay
             $$('ma-datagrid-pagination .total').then(function (totalElements) {
-                expect(totalElements[0].getText()).toBe('1 - 5 on 5');
+                expect(totalElements[0].getText()).toBe('1 - 4 on 4');
```

Comment:
I don't know how it's possible to find 5 records, see http://ng-admin.marmelab.com/#/comments/list?search={%22q%22:%22be%22}&page=1

But if I search for `b` instead of `be`, 5 record are found, maybe a coincidence, but this is the only explaination I see, protractor wasn't wait for the `e` with the old basecode.